core-bundle/src/Doctrine/Backup/Dumper.php

```diff
@@ -61,7 +60,7 @@ private function dumpViews(AbstractSchemaManager $schemaManager, AbstractPlatfor
     {
         foreach ($schemaManager->listViews() as $view) {
             yield sprintf('-- BEGIN VIEW %s', $view->getName());
-            yield $platform->getCreateViewSQL($view->getQuotedName($platform), $view->getSql()).';';
```

Comment:
We can just make 2 queries out of it, as it's non persistent data anyway? Like so?

```php
            yield $platform->getDropViewSQL($view->getQuotedName($platform)).';';
            yield $platform->getCreateViewSQL($view->getQuotedName($platform), $view->getSql()).';';
```

Reply:
This will fail if the view does not exist.

Reply:
Right :) 👍 then